google/cloud/aiplatform/models.py

```diff
@@ -354,6 +353,35 @@ def _unallocate_traffic(
 
         return new_traffic_split
 
+    @staticmethod
+    def _validate_explanation_args(
+        explanation_metadata: Optional["aiplatform.explain.ExplanationMetadata"] = None,
```

Comment:
It seems like the forward reference is not required here. Can this be:
`Optional[explain.ExplanationMetadata]`.

Same for the line below.